File: app/src/main/java/com/lianpos/devfoucs/contacts/model/PhoneBean.java
package com.lianpos.devfoucs.contacts.model;


import com.mcxtzhang.indexlib.IndexBar.bean.BaseIndexPinyinBean;

/**
 * Created by wangshuai .
 * Date: 17/10/31
 */

public class PhoneBean {

    private String phone;//城市名字
    private boolean isTop;//是否是最上面的 不需要被转化成拼音的

    public PhoneBean() {
    }

    public PhoneBean(String phone) {
        this.phone = phone;
    }

    public String getPhone() {
        return phone;
    }

    public PhoneBean setPhone(String phone) {
        this.phone = phone;
        return this;
    }

    public boolean isTop() {
        return isTop;
    }

    public PhoneBean setTop(boolean top) {
        isTop = top;
        return this;
    }
}

File: app/src/main/java/com/lianpos/devfoucs/contacts/model/PhoneBean.kt
package com.lianpos.devfoucs.contacts.model


import com.mcxtzhang.indexlib.IndexBar.bean.BaseIndexPinyinBean

/**
 * Created by wangshuai .
 * Date: 17/10/31
 */

class PhoneBean {

    private var phone: String? = null//城市名字
    private var isTop: Boolean = false//是否是最上面的 不需要被转化成拼音的

    constructor() {}

    constructor(phone: String) {
        this.phone = phone
    }

    fun getPhone(): String? {
        return phone
    }

    fun setPhone(phone: String): PhoneBean {
        this.phone = phone
        return this
    }

    fun isTop(): Boolean {
        return isTop
    }

    fun setTop(top: Boolean): PhoneBean {
        isTop = top
        return this
    }
}
